module/apmgopgv10/doc.go

```diff
@@ -0,0 +1,21 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// +build go1.11
```

Comment:
Tests are failing on 1.11 and 1.12:

> ../../../../../pkg/mod/github.com/go-pg/pg/v10@v10.7.3/orm/table.go:40:40: undefined: sql.NullTime

Looks like the build tag should be `go1.14` (here and elsewhere in this package)

Although the tests pass on 1.13 currently, go-pg v10 is only testing 1.14+: https://github.com/go-pg/pg/blob/v10/.travis.yml. So I think we should just set it to 1.14 too, in case there's some change in go-pg that breaks 1.13 later.

Reply:
Done